pyxnat/tests/resources_test.py

```diff
@@ -80,63 +81,64 @@ def test_multi_create():
 
 #def test_share_experiment():
-#    assert not central.select('/projects/nosetests2/subjects/%(sid)s'
+#    assert not central.select('/projects/nosetests32/subjects/%(sid)s'
 #                          '/experiments/%(eid)s'%_id_set1
 #                          ).exists()
-#    expe_1.share('nosetests2')
-#    assert central.select('/projects/nosetests2/subjects/%(sid)s'
+#    expe_1.share('nosetests32')
+#    assert central.select('/projects/nosetests32/subjects/%(sid)s'
 #                          '/experiments/%(eid)s'%_id_set1
 #                          ).exists()
-#    assert set(expe_1.shares().get()) == set(['nosetests', 'nosetests2'])
+#    assert set(expe_1.shares().get()) == set(['nosetests3', 'nosetests32'])
 
 #def test_share_assessor():
-#    assert not central.select('/projects/nosetests2/subjects/%(sid)s'
+#    assert not central.select('/projects/nosetests32/subjects/%(sid)s'
 #                              '/experiments/%(eid)s/assessors/%(aid)s'%_id_set1
 #                              ).exists()
-#    asse_1.share('nosetests2')
-#    assert central.select('/projects/nosetests2/subjects/%(sid)s'
+#    asse_1.share('nosetests32')
+#    assert central.select('/projects/nosetests32/subjects/%(sid)s'
 #                          '/experiments/%(eid)s/assessors/%(aid)s'%_id_set1
 #                          ).exists()
-#    assert set(asse_1.shares().get()) == set(['nosetests', 'nosetests2'])
+#    assert set(asse_1.shares().get()) == set(['nosetests3', 'nosetests32'])
 
 #def test_unshare_assessor():
-#    asse_1.unshare('nosetests2')
-#    assert not central.select('/projects/nosetests2/subjects/%(sid)s'
+#    asse_1.unshare('nosetests32')
+#    assert not central.select('/projects/nosetests32/subjects/%(sid)s'
 #                              '/experiments/%(eid)s/assessors/%(aid)s'%_id_set1
 #                              ).exists()
-#    assert asse_1.shares().get() == ['nosetests']
+#    assert asse_1.shares().get() == ['nosetests3']
 
 #def test_unshare_experiment():
-#    expe_1.unshare('nosetests2')
-#    assert not central.select('/projects/nosetests2/subjects/%(sid)s'
+#    expe_1.unshare('nosetests32')
+#    assert not central.select('/projects/nosetests32/subjects/%(sid)s'
 #                          '/experiments/%(eid)s'%_id_set1
 #                          ).exists()
-#    assert expe_1.shares().get() == ['nosetests']
+#    assert expe_1.shares().get() == ['nosetests3']
 
 #def test_unshare_subject():
-#    subj_1.unshare('nosetests2')
-#    assert not central.select('/projects/nosetests2/subjects/%(sid)s'%_id_set1
+#    subj_1.unshare('nosetests32')
+#    assert not central.select('/projects/nosetests32/subjects/%(sid)s'%_id_set1
 #                              ).exists()
-#    assert subj_1.shares().get() == ['nosetests']
+#    assert subj_1.shares().get() == ['nosetests3']
 
 def test_put_file():
     local_path = os.path.join(_modulepath, 'hello_xnat.txt')
+    f = subj_1.resource('test').file('hello.txt')
```

Comment:
Suggestion: reuse `f` later on in the function to avoid redundancy and shorten length and complexity of code:
```
f = subj_1.resource('test').file('hello.txt')
f.put(local_path)
assert f.exists()
assert int(f.size()) == os.stat(local_path).st_size	                                                ```

Reply:
Fixed in 5025215.